subt/artf_node.py

```diff
@@ -27,6 +28,26 @@
 }
 
 
+def check_results(result, result_cv):
+    ret = []
+    if len(result) == 0 or len(result_cv) == 0:
+        return ret
+    for r_cv in result_cv:
+        name_cv, score_cv, bbox = r_cv
+        bbox_x1, bbox_y1, bbox_x2, bbox_y2 = bbox
+        for r in result:
+            name, points = r
+            if name != name_cv:
+                continue
+            x = np.array([p[0] for p in points])
+            y = np.array([p[1] for p in points])
+            x_in_bbox = (x > bbox_x1) & (x < bbox_x2)  # arr of boolean values
```

Comment:
I would expect `and` instead of `&`. Same on the next line.

Should the bounds be exclusive, or inclusive?

Reply:
> I would expect and instead of &. Same on the next line.

No, there have to be `&`.

> Should the bounds be exclusive, or inclusive?

If i understand correctly the current state is exclusive. I think it should be like that.

Reply:
Right, sorry, with numpy arrays, we need `np.logical_and` instead of the python `and`.